src/imagespec_fast_builder/_image_builder.py

```diff
@@ -199,6 +202,11 @@ def create_docker_context(image_spec: ImageSpec, tmp_dir: Path):
     else:
         python_version = f"{sys.version_info.major}.{sys.version_info.minor}"
 
+    if image_spec.entrypoint is None:
```

Comment:
Just so we support older versions of `flytekit<1.13.0`:

```suggestion
    if getattr(image_spec, "entrypoint", None) is None:
```


(`image_spec.entrypoint` was introduced in `1.13.0`)